server/accounts.go

```diff
@@ -1741,12 +1741,12 @@ func (a *Account) addServiceImport(dest *Account, from, to string, claim *jwt.Im
 	if claim != nil {
 		share = claim.Share
 	}
-
+	if !share {
```

Comment:
Help me understand: is it that share should be true if isSysAcc, but if there is a claim, then share becomes whatever claim.Share is?
If so, would not it be simpler to have:
```
// Turn on sharing by default if importing from system services.
share := isSysAcc
if claim != nil {
    share = claim.Share
}
si := &serviceImport{...
```

Reply:
the problem was that if share was true an disSysAcc was false, the earlier code would have resulted in false.
yes, will change to your suggestion